src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import core.basesyntax.interfaces.AreaCounter;
+import core.basesyntax.interfaces.Drawable;
+
+public abstract class Figure implements Drawable, AreaCounter {
+    public abstract void draw();
+
+    public abstract double area();
+
```

Comment:
you don't need to create abstract method here because you have `Drawable, AreaCounter` for this purpose
`Figure` should have `color`
```suggestion
    private String color;
    
    public Figure(String color) {
        this.color = color;
    }
```